kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/documentation/DocumentationLinkItem.less

```
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[data-i18n-prefix="DocumentationLinkItem."] {
+
+  .documentation-links {
+    height: 60px;
+    overflow-y: scroll;
+    overflow-x: hidden;
+  }
+
+  .col-md-10, col-md-1 {
```

I think that we can remove the `col-md-1`:
```suggestion
  .col-md-10 {
```
(since we have a missing dot, this "class" is not applying any style).